### PR TITLE
Switch to use swift-argument-parser and add a --verbose switch

### PR DESCRIPTION
**Pull request by tamc**
This PR switches to using the swift-argument-parser package and then adds a --verbose flag to start-server subcommand

The --verbose flag causes the server to print the command that is being run and the output of that command

### CODE REVIEW

**Comment by ctreffs** [comment]
Thanks for contributing this, I originally started with ArgumentParser, but back then it was not compatible enough. 
Let's give it another try now 👍

File: Sources/SimctlCLI/ListDevices.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
+//
+//  SimctlCLI.swift
```
Comment (ctreffs) [inline Sources/SimctlCLI/ListDevices.swift:2]: I know it's a nitpick, but would be nice to rename the header comment to be equal to the file name.

File: Sources/SimctlCLI/StartServer.swift [hunk 2]
```diff
@@ -0,0 +1,85 @@
+//
+//  SimctlCLI.swift
```
Comment (ctreffs) [inline Sources/SimctlCLI/StartServer.swift:2]: Same here :-)

**Review by ctreffs — CHANGES_REQUESTED** [review]
Very nice PR, thank you for your contribution!
I just hat two nitpicks, other then that I would merge it.

One more thing - to make the PR complete, it would be nice, if you generate the new version of the binary `SimctlCLI`, that can be achieved by calling `make buildSimctlCLI`.
That updates the executable in `bin/` which is necessary to run the tests.
Thank you :-)

**Comment by scott-mcalister** [comment]
Hey @tamc, are you planning on wrapping up this request?

**Comment by rsmcalister** [comment]
Hey @tamc, are you planning on wrapping up this request?

**Comment by ctreffs** [comment]
That would be great. Please do 👍🏻

**Comment by ctreffs** [comment]
Closed by https://github.com/ctreffs/SwiftSimctl/pull/12

**Pull request closed without merging**